apps/meteor/tests/e2e/channel-management.spec.ts

```diff
@@ -57,69 +57,81 @@ test.describe.serial('channel-management', () => {
 		await expect(page.getByRole('button', { name: 'Start call' })).toBeFocused();
 	});
 
-	// FIXME: bad assertion
-	test('should add "user1" to "targetChannel"', async () => {
+	test('should add user1 to targetChannel', async () => {
 		await poHomeChannel.sidenav.openChat(targetChannel);
 		await poHomeChannel.tabs.btnTabMembers.click();
 		await poHomeChannel.tabs.members.showAllUsers();
 		await poHomeChannel.tabs.members.addUser('user1');
 
-		await expect(poHomeChannel.toastSuccess).toBeVisible();
+		await expect(poHomeChannel.getSystemMessageByText('added user1')).toBeVisible();
 	});
 
-	// FIXME: bad assertion
-	test('should create invite to the room', async () => {
-		await poHomeChannel.sidenav.openChat(targetChannel);
-		await poHomeChannel.tabs.btnTabMembers.click();
-		await poHomeChannel.tabs.members.inviteUser();
-
-		await expect(poHomeChannel.toastSuccess).toBeVisible();
-	});
-
-	test.fixme('should mute "user1"', async () => {
+	test('should mute user1', async () => {
 		await poHomeChannel.sidenav.openChat(targetChannel);
 		await poHomeChannel.tabs.btnTabMembers.click();
 		await poHomeChannel.tabs.members.showAllUsers();
 		await poHomeChannel.tabs.members.muteUser('user1');
+
+		await expect(poHomeChannel.getSystemMessageByText('muted user1')).toBeVisible();
```

Comment:
Same here, we could ensure that the user can't actually write in the channel after being muted